globalmangrovewatch/gmw-users
# Feature: 10.7

10.7 has two parts:
1) Grouped checked items (10.7)
2) Additional info (10.7a)

10.7a only appears when at least one item in 10.7 is checked

To test:
- Create a new site
- Go to section 3
- Select items in 3.1 (these appear as linked aims in 10.7a)
- Add section 10 monitoring form
- Check items in 10.7
- fill additional details in 10.7a

note - there is no delete warning at the moment. 
So if an item is unchecked and information in 10.7a is filled out, it will be lost. I want to make a separate task for this
